index.bs

```diff
@@ -270,6 +275,8 @@ An attribution trigger is a [=struct=] with the following items:
 :: A [=byte sequence=].
 : <dfn>debug reporting enabled</dfn>
 :: A [=boolean=].
+: <dfn>aggregation coordinator</dfn>
+:: Null or an [=aggregation coordinator=].
```

Comment:
What's the benefit of null vs an implementation-defined default?

Reply:
Not particular, just to avoid defining another implementation-defined default, but I don't feel strongly. Wdyt?

Reply:
I think I'd slightly lean towards not null (but I don't feel strongly). That being said, if we keep null, it probably makes sense to explicitly call out that null implies use a default in "obtain the public key for encryption".

Reply:
Updated to call out that a default may be used if it is null. Please let me know if anyone feels strongly to use not null. Thanks.